### PR TITLE
AnimalType Service, Controller, Tests added

### PR DESCRIPTION
**Pull request by kaboo21**

### CODE REVIEW

File: VetClinic.BLL.Tests/Services/PetServiceTests.cs [hunk 1]
```diff
@@ -26,47 +26,53 @@ public PetServiceTests()
...
+            // Arrange
             _petRepository.Setup(x => x.GetAsync(null, null, null, true).Result)
                 .Returns(PetFakeData.GetPetFakeData());
-
+            // Act
             IList<Pet> tempPets = await _petServise.GetPetsAsync();
 
+            // Assert
             Assert.NotNull(tempPets);
             Assert.Equal(10,tempPets.Count);
```
Comment (Serge1212) [inline VetClinic.BLL.Tests/Services/PetServiceTests.cs:37]: dont use magic numbers in assert, move them to variables

File: VetClinic.BLL.Tests/Services/PetServiceTests.cs [hunk 2]
```diff
@@ -26,47 +26,53 @@ public PetServiceTests()
...
-       
             _petRepository.Setup(x => x.GetFirstOrDefaultAsync(x => x.Id == id, null, false).Result)
                 .Returns(pets.FirstOrDefault(x => x.Id == id));
-
+            
+            // Act
             var pet = await _petServise.GetByIdAsync(id);
 
+            // Assert
             Assert.Equal("Lord10", pet.Name);
```
Comment (Serge1212) [inline VetClinic.BLL.Tests/Services/PetServiceTests.cs:53]: move expected value to variable

File: VetClinic.WebApi.Tests/Controllers/PetsControllerTests.cs [hunk 3]
```diff
@@ -149,6 +152,7 @@ public void CanGetAllPets()
         [Fact]
         public void CanGetPetById()
         {
+            // Arrange
             var pets = GetTestPets().AsQueryable();
             var id = 10;
             var name = "Lord10";
```
Comment (Serge1212) [inline VetClinic.WebApi.Tests/Controllers/PetsControllerTests.cs:158]: why would you need this variable?

Comment (kaboo21) [reply VetClinic.WebApi.Tests/Controllers/PetsControllerTests.cs:158]: what variable?  

this?=>var name = "Lord10";

Assert.Equal(name, model.Name);

Comment (Serge1212) [reply VetClinic.WebApi.Tests/Controllers/PetsControllerTests.cs:158]: yes

**Review by Serge1212 — APPROVED** [review]

**Review by StanislavKudriavtsev42 — APPROVED** [review]

**Pull request merged**